setup.py

```diff
@@ -48,7 +48,7 @@ def _plugin_version() -> str:
     packages=find_namespace_packages(include=["dbt", "dbt.*"]),
     include_package_data=True,
     install_requires=[
-        "dbt-common<1.0",
+        "dbt-common~=0.1.0",
```

Comment:
Per our recent discussion on `~=`, we want to move to the equivalent `>=,<` syntax. Hence this should be `dbt-common>=0.1.0,<0.2.0`.